src/AWSS3.jl

```diff
@@ -783,7 +798,9 @@ function s3_list_objects(
         end
     end
 end
-s3_list_objects(a...; kw...) = s3_list_objects(global_aws_config(), a...; kw...)
+function s3_list_objects(bucket, path_prefix=""; kw...)
+    return s3_list_objects(global_aws_config(), bucket, path_prefix; kw...)
+end
```

Comment:
Just for consistency,

```suggestion
function s3_list_objects(bucket, path_prefix=""; kwargs...)
    return s3_list_objects(global_aws_config(), bucket, path_prefix; kwargs...)
end
```